functions/src/modules/SE.functions.ts

```diff
@@ -16,15 +16,13 @@ export const uploadProcessing = functions.storage.object()
 
     const filePath = object.name;
     
-    if (object.name.slice(-4).toLowerCase() !== 'flac') {
-        console.error("Something is wrong with the file extension!")
-        return -1;
-    }
-
     const filePathRegex = /sound-editing\/restored\/(\w+)\/(\w+)/g;
```

Comment:
`\w` is not good here. Test for task ID and the file extension in this regex.

Reply:
Ok, but the grouping still needs to be based on the whole name `list-serial.flac` .. right ?